### PR TITLE
Some Forge Foundry Unit Tests

### PR DESCRIPTION
**Pull request by kamikazebr**
- [x] Test for overflow and creation pair

### CODE REVIEW

File: scripts/deployerCaller.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import * as zksync from 'zksync-web3'
```
Comment (rperez89) [inline scripts/deployerCaller.ts:1]: i think we can totally remove this file, was just a testing one

**Review by rperez89 — APPROVED** [review]

File: scripts/deployerCaller.ts (hunk 1 above)
Comment (kamikazebr) [reply scripts/deployerCaller.ts:1]: You mean the whole file or the import?

Comment (rperez89) [reply scripts/deployerCaller.ts:1]: the deployerCaller file itself

Comment (kamikazebr) [reply scripts/deployerCaller.ts:1]: I'm using it to call the function deploy in fact after that get deployed. I'll migrate it to foundry later on.

**Pull request merged**